### DISCUSSION TITLE
local path on Windows

### DISCUSSION THREAD

**Discussion by khm1234** [grafana/grafana-csv-datasource#112] [Q&A]
I must be missing something obvious -- how do I specify a path to a CSV on a windows machine?  Or maybe I didn't properly configure to allow local files?   I "add data source" and get to the screen shown below.   

![Capture](https://user-images.githubusercontent.com/87500807/125856515-82cda3b3-4a55-40ae-989f-ff80ac3983cc.JPG)

**Comment by Thanitorial** [comment]
hi there

if this has been fixed/answered then please ignore but for those whom are strugling with the same thing as i was here i may have a fix for you below \/ hope this will help you out in atempting to get this to work for you 😄 


try doing the following to get it to work.

after i followed the steps in the docs i came to the same issue as you. the grafana service needs to be restarted in the services app( can be done by: 
firstly 
making sure the following is added to the very bottom of the default.ini file in conf of grafana (can be found here : C:\Program Files\GrafanaLabs\grafana\conf) 
[plugin.marcusolsson-csv-datasource]
allow_local_mode = true

then do the following:
win key -> search the following "services" and open app -> scroll to find grafana-> right click -> restart(make sure all instances of grafana are closed or this may not do what is intended :D ) 

this then led me to open a new window and make a new data source where i did the following
![Screenshot 2021-12-08 214556](https://user-images.githubusercontent.com/64614575/145289554-5e032697-2fa5-4a60-8411-0ebdf7d08c04.png)
and clicked test and save, this is where you define the file path to your file and voila it be fixed 👍
